evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -576,36 +583,7 @@ def test_multi_format_creation(X_y_binary):
     assert not clf.feature_importance.isnull().all().all()
 
 
-def test_multiple_feature_selectors(X_y_binary):
```

Comment:
I'm not really sure what exactly this was testing... just that we could have multiple feature selectors defined in a component graph? I deleted because I'm not sure how this differs from simply testing that we have duplicate components in a pipeline, but feel free to point out something I'm missing!